packages/app/src/main.ts

```diff
@@ -3,7 +3,6 @@ import { createApp } from 'vue';
 import { socket as feathersSocket } from '@/lib/feathers';
 
 import App from './App.vue';
-import './index.css';
```

Comment:
Use this line again.

Reply:
fixed in 4bdb939bf1512fcb307e91af91ffd2fc6a71087f